examples/src/main/scala/PekkoHttpExample.scala

```diff
@@ -0,0 +1,191 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import cats.effect.Async
+import cats.effect.IO
+import cats.effect.IOApp
+import cats.effect.IOLocal
+import cats.effect.Resource
+import cats.effect.Sync
+import cats.effect.std.Random
+import cats.effect.syntax.temporal._
+import cats.mtl.Local
+import cats.syntax.applicative._
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import io.opentelemetry.api.GlobalOpenTelemetry
+import io.opentelemetry.context.{Context => JContext}
+import io.opentelemetry.instrumentation.annotations.WithSpan
+import org.apache.pekko.actor.ActorSystem
+import org.apache.pekko.http.scaladsl.Http
+import org.apache.pekko.http.scaladsl.model.HttpRequest
+import org.apache.pekko.http.scaladsl.model.StatusCodes.OK
+import org.apache.pekko.http.scaladsl.server.Directives._
+import org.apache.pekko.http.scaladsl.server.Route
+import org.apache.pekko.util.ByteString
+import org.typelevel.otel4s.Attribute
+import org.typelevel.otel4s.java.OtelJava
+import org.typelevel.otel4s.java.context.Context
+import org.typelevel.otel4s.java.instances._
+import org.typelevel.otel4s.trace.Tracer
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+/** This example relies on the OpenTelemetry Java agent. To make it work, add
+  * the following settings to your build:
+  *
+  * add `sbt-javaagent` dependency to the `plugins.sbt`:
+  *
+  * {{{
+  * addSbtPlugin("com.github.sbt" % "sbt-javaagent" % "0.1.8")
+  * }}}
+  *
+  * update definition of a project in the `build.sbt`:
+  *
+  * {{{
+  * .enablePlugins(JavaAgent)
+  * .settings(
+  *   libraryDependencies ++= Seq(
+  *     "org.typelevel"                   %% "otel4s-java"                               % "0.3.0-RC2",
+  *     "org.apache.pekko"                %% "pekko-stream"                              % "1.0.1",
+  *     "org.apache.pekko"                %% "pekko-http"                                % "1.0.0",
+  *     "io.opentelemetry.instrumentation" % "opentelemetry-instrumentation-annotations" % "1.31.0",
+  *     "io.opentelemetry"                 % "opentelemetry-exporter-otlp"               % "1.31.0" % Runtime,
+  *     "io.opentelemetry"                 % "opentelemetry-sdk-extension-autoconfigure" % "1.31.0" % Runtime
+  *   )
+  *   run / fork := true,
+  *   javaOptions += "-Dotel.java.global-autoconfigure.enabled=true",
+  *   javaOptions += "-Dotel.service.name=pekko-otel4s",
+  *   javaAgents += "io.opentelemetry.javaagent" % "opentelemetry-javaagent" % "1.31.0" % Runtime
+  * )
+  * }}}
+  */
+object PekkoHttpExample extends IOApp.Simple {
+
+  def run: IO[Unit] =
+    IOLocal(Context.root).flatMap { implicit ioLocal: IOLocal[Context] =>
+      implicit val local: Local[IO, Context] = localForIoLocal
+      val otelJava: OtelJava[IO] = OtelJava.local(GlobalOpenTelemetry.get())
```

Comment:
IIRC, otel recommends not using the global instance. is there another instance we can use relatively easily in examples? because examples invariably get copied into non-example code and end up in production

Reply:
An instance can be created `AutoConfiguredOpenTelemetrySdk.builder().build()`. It creates an autoconfigured instance and does not make it global.

But in this particular example, it must be global because the instance is created by the Otel javaagent upon JVM startup.

Reply:
> IIRC, otel recommends not using the global instance.

But other examples could be updated